b2sdk/filter.py

```diff
@@ -0,0 +1,53 @@
+######################################################################
+#
+# File: b2sdk/filter.py
+#
+# Copyright 2024 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+from __future__ import annotations
+
+import fnmatch
+from enum import Enum
+from typing import NamedTuple, Sequence
+
+
+class FilterType(Enum):
+    INCLUDE = "include"
+    EXCLUDE = "exclude"
+
+
+class Filter(NamedTuple):
+    type: FilterType
+    pattern: str
+
+
+def include(pattern: str) -> Filter:
```

Comment:
Since these functions are exported in flat `_v3` namespace, `include` feels like it might not be descriptive enough. Would it make sense to implement `include` and `exclude` as class methods of `Filter`? Or rename them?

Although I can see why you'd want to avoid longer names here.

Reply:
I agree. I've moved them under `Filter` as class methods.

We do want them to be shorter, but their use case pretty low, given the fact that we support wildcards in `folderName` as well.